src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,18 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.shredding.serialization.total.documents")
+  String jsonShreddingSerializedTotalDocuments();
+
+  @NotBlank
+  @WithDefault("json.shredding.serialization.size")
+  String jsonShreddingSerializedSize();
+
+  @NotBlank
+  @WithDefault("json.shredding.serialization.performance")
+  String jsonSerializationPerformanceMetricsName();
+
   @NotBlank
   @WithDefault("command.processor.process")
   String metricsName();
```

Comment:
I know this is existing code, but I think name here is way too generic. It does not matter for actual metrics name selection, but I think it should be changed to something more meaningful. Like:

``
   String commandProcessorMetricsName();
```

Reply:
> I know this is existing code, but I think name here is way too generic. It does not matter for actual metrics name selection, but I think it should be changed to something more meaningful. Like:
> 
> ```
>    String commandProcessorMetricsName();
> ```

I think we should leave this alone, we've already got downstream metrics processing and dashboards built off of the current name.